src/prx/test/test_helpers.py

```diff
@@ -156,3 +206,131 @@ def test_sagnac_effect():
     # millimeter accuracy should be sufficient
     tolerance = 1e-3
     assert np.max(np.abs(sagnac_effect_computed - sagnac_effect_reference)) < tolerance
+
+def test_compute_inter_constellation_bias_from_rinex3(input_for_test):
+    # filepath towards RNX3 NAV file
+    path_to_rnx3_nav_file = converters.anything_to_rinex_3(
+        input_for_test["nav"]
+    )
+
+    # Parse the RNX3 NAV file
+    computed_time_system_corr_dict= helpers.parse_time_syst_corr_from_rinex_nav_file([path_to_rnx3_nav_file])
+
+    t = 3600.0   #For an single EPOCH the value of week of seconds for 2023-01-01 01 00 00
+    w = 2243     #weeks
+    computed_icb_dict = helpers.compute_icb_all_constellations(computed_time_system_corr_dict, t, w)
+    # Manually defined ICB dictionary values
+    manual_icb_dict = {
```

Comment:
What is the formula to obtain these values?

Reply:
For Galileo, I do the following manual computatation:
 `(4.7439243644E-09 + 4.884981308E-15 * (t - 0 + 86400*7*(w-2243)))*constants.cGpsSpeedOfLight_mps`, which is equal to `1.4274648757625577` m, 

The value in the  `manual_icb_dict["E"]`, which is equal to `-0.73162253` m

Reply:
When I do the manual computation for Galileo, copying the numbers that are in the RNX header, I have:
`(4.7439243644E-09 + 4.884981308E-15 * (t - 0 + 86400*7*(w-2243)) - (-1.8626451492E-09 + 6.217248938E-15 * (t - 233472 + 86400*7*(w-2243))))*constants.cGpsSpeedOfLight_mps`
which is equal to **2.414326664685148** m.

While the value in `manual_icb_dict["E"]` is **-0.73162253** m

Reply:
Actually the GAUT values are used to compute the Inter constellation bias of Galileo to the GPS instead of using the GAGP so the revised values of time system corr dictionary for the GAUT are A0: -9.3132257E-10 A1: 8.8817841E-16 T: 518400 W: 2242 therefore the from the above formula Time System correction for Galileo = (-9.3132257E-10 + 8.8817841E-16 * (3600 - 518400 + 604800 * (2243 - 2242 ))) =  -8.513865131 seconds and Time system correction for reference = -3.2918165E-9 sec. 
ICB = time system corr reference - time system corr GAL = (-2.440430026E-9 sec) * speed of light = -0.7322 m

Reply:
Yes, my mistake...
This seems correct then.
Still add a comment to show explicitly the manual computation.